### ISSUE TITLE
docs: Access to replica of database

### ISSUE THREAD

**Issue by davidyuk** [aeternity/aepp-middleware#159]
> You can start off by priming with our database--we expose a read-only replica of the PostgreSQL database which we also expose.
https://github.com/aeternity/aepp-middleware/blob/develop/docs/middleware-guide.md#installation-or-not

where it is published?

**Comment by jsnewby** [comment]
Yes this was trickier than anticipated. Reference removed until we can fix this.
